src/components/Layout.res

```diff
@@ -195,21 +159,7 @@ let make = () => {
                   ReactEvent.Mouse.preventDefault(event)
                   clearAndPush(j`/#leaderboards/monthly-contribution`)
                 }}>
-                {"LEADERBOARDS"->restr}
-              </a>,
-          },
-          {
-            shouldDisplay: true,
-            shouldDisplayMobile: true,
-            component: (closeModal, _) =>
-              <a
-                className=Styles.navListText
-                onClick={event => {
-                  closeModal()
-                  ReactEvent.Mouse.preventDefault(event)
-                  clearAndPush(j`/#dao`)
-                }}>
-                {"DAO"->restr}
```

Comment:
Goodbye dao - will add back when we get that functionality.

Should never use the word "DAO" again though, IMO. I have no idea what it should say but something along the lines of "governance" or "Community"

Reply:
Hmm none are fully clicking but we can cross that bridge when we get to it. 
I like the shortness characteristics of DAO yet agree it's not clear to the everyday person.

Reply:
Yeah, we'll cross that bridge when it comes